server/src/test/java/org/opensearch/index/get/GetResultTests.java

```diff
@@ -220,6 +224,22 @@ public void testEqualsAndHashcode() {
         );
     }
 
+    public void testFomXContentEmbeddedFoundParsingException() throws IOException {
```

Comment:
Oh, found a typo here, the test method name should be `testFromXContent...`,  the original PR in main branch also has this problem, @imvtsl could you fix this in another PR?

Reply:
Thanks @gaobinlong! I'll merge this backport and the typo can be fixed in a follow up.

Reply:
@gaobinlong Thank you for pointing out! I will fix it very soon.